README.md

```diff
@@ -260,19 +261,45 @@ It is also straightforward to have a UIKit controller driven off of this store.
 Once we are ready to display this view, for example in the app's entry point, we can construct a store. This can be done by specifying the initial state to start the application in, as well as the reducer that will power the application:
 
 ```swift
+import ComposableArchitecture
+
 @main
 struct MyApp: App {
   var body: some Scene {
-    FeatureView(
-      store: Store(
-        initialState: Feature.State(),
-        reducer: Feature()
+    WindowGroup {
+      FeatureView(
+        store: Store(
+          initialState: Feature.State(),
+          reducer: Feature()
+        )
       )
-    )
+    }
   }
 }
 ```
 
+You may also find that the HTTP API request to numbersapi.com is blocked as only HTTPS connections are permitted.  If this happens you can add an exception to the project settings to allow HTTP requests just for the numbersapi.com domain.
+
+```
+<key>NSAppTransportSecurity</key>
+<dict>
+  <key>NSAllowsArbitraryLoads</key>
+  <false/>
+  <key>NSExceptionDomains</key>
+  <dict>
+    <key>numbersapi.com</key>
+    <dict>
+      <key>NSIncludesSubdomains</key>
+      <true/>
+      <key>NSTemporaryExceptionAllowsInsecureHTTPLoads</key>
+      <true/>
+      <key>NSTemporaryExceptionMinimumTLSVersion</key>
+      <string>TLSv1.1</string>
+    </dict>
+  </dict>
+</dict>
+```
+
```

Comment:
I think we should probably host our own demo from an HTTPS location to avoid this confusion. Sorry @sillygoose!

For now, though, I think we probably shouldn't bog down the README with an issue that isn't library-specific.

```suggestion
```